src/client/opamClient.ml

```diff
@@ -153,8 +154,9 @@ let details_of_package_regexps t packages ~exact_name ~case_sensitive regexps =
 
 let with_switch_backup command f =
```

Comment:
Now the switch state is directly stored in the export format, so exporting the switch amounts to just backing up the `~/.opam/<switch>/state` file: much cheaper than it used to be!